src/transaction/Transaction.js

```diff
@@ -800,6 +810,11 @@ export default class Transaction extends Executable {
         const isSingleSignature = signature instanceof Uint8Array;
         const isArraySignature = Array.isArray(signature);
 
+        if (this.getRequiredChunks() > 1) {
+            throw new Error(
+                "Add signature is not supported for chunked transactions",
+            );
+        }
```

Comment:
What if you have a ChunkedTransacrtion with one chunk?

Reply:
Just added an integration test for this case. I think it works properly as of right now but it will probably going to get thoroughly tested in the new version when we have to add the signature functionality for file append too.

Reply:
Where is this integration test? Couldn't find...

Reply:
my bad nvm

Reply:
`FileAppendIntegrationTest.js` - "should be able to sign transaction with 1 chunk"